frp_deploy.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#Set flags for hot update.
+
+frps_FLAG=`ps -ef |grep -w frps|grep -v grep|wc -l`
+frpc_FLAG=`ps -ef |grep -w frpc|grep -v grep|wc -l`
+
+#Force stop services before upgrade.
+
+systemctl stop frps
```

Comment:
`deploy.sh` may not need to start or stop service now. We can just display start and stop command after successful installed.